packages/trpc/server/routers/viewer/teams/listMembers.handler.ts

```diff
@@ -14,6 +14,13 @@ type ListMembersOptions = {
 
 export const listMembersHandler = async ({ ctx, input }: ListMembersOptions) => {
   const { prisma } = ctx;
+  const { isOrgAdmin } = ctx.user.organization;
+  const hasPermsToView = ctx.user.organization.isPrivate ? isOrgAdmin : true;
```

Comment:
Nit: Do we need the ternary?

```suggestion
  const hasPermsToView = !ctx.user.organization.isPrivate || isOrgAdmin;
```

Reply:
Good spot